modules/gdscript/gdscript_functions.cpp

```diff
@@ -124,7 +126,7 @@ const char *GDScriptFunctions::get_func_name(Function p_func) {
 		"get_stack",
 		"instance_from_id",
 		"len",
-		"is_instance_valid",
+		"is_instance_valid"
```

Comment:
I believe that it's preferred to leave trailing commas because of version control working line-by-line.